contracts/cheddar_5x5_tic_tac_toe/src/board.rs

```diff
@@ -47,46 +47,34 @@ impl Board {
     /// there is two players with different AccountId's and
     /// with different random given pieces
     /// accounts check in `Game.create_game`
-    pub fn new(game_id: GameId, player_1: &Player, player_2: &Player) -> Self {
-        assert_ne!(
-            player_1.piece, player_2.piece,
-            "players have same pieces: {:?}",
-            player_1.piece
-        );
+    pub fn new(game_id: GameId) -> Self {
```

Comment:
let's update the documentation (README) saying that `O` always starts, and we select the starting player using NEAR random mechanism.